base/linalg/transpose.jl

```diff
@@ -0,0 +1,334 @@
+immutable Transpose{T,S<:AbstractMatrix} <: AbstractMatrix{T}
+    data::S
+    conjugated::Bool
+end
+
+# temp definitions while Transpose is not subtype of AbstractMatrix
+# Base.eltype{T}(A::Transpose{T}) = T
+
+# end temp
+
+Base.size(A::Transpose) = size(A.data, 2), size(A.data, 1)
+function Base.size(A::Transpose, i::Integer)
```

Comment:
I often forget this myself, but `size(A, i)` works automatically if you define `size(A)` now.